mbtiles/src/mbtiles.rs

```diff
@@ -690,10 +861,17 @@ pub async fn attach_hash_fn(conn: &mut SqliteConnection) -> MbtResult<()> {
     Ok(())
 }
 
+fn webmercator_to_wgs84(x: f64, y: f64) -> [f64; 2] {
```

Comment:
arrays are usually not good in such cases --  a better pattern is to return as you did before - `-> (f64, f64)`. Arrays tend to mean "some number of identical elements", whereas tuples indicate that the elements are positional and might have different types.  Here, it just happened to be the same type.